problems/[230907] 92. Reverse Linked List ||.js.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Definition for singly-linked list.
+ * function ListNode(val, next) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.next = (next===undefined ? null : next)
+ * }
+ */
+/**
+ * @param {ListNode} head
+ * @param {number} left
+ * @param {number} right
+ * @return {ListNode}
+ */
+var reverseBetween = function (head, left, right) {
+  if (left === right) return head;
+
+  let copyList = head;
+  let i = 1;
+  const valuesArr = [];
+
+  while (copyList && i <= right) {
```

Comment:
문제 제약조건이 있어서 이렇게만 작성해도 동일하게 동작할 것 같아요
```suggestion
  while (i <= right) {
```